Angular/Angular guidelines and best practices/DatX data store.md

```diff
@@ -0,0 +1,206 @@
+This chapter covers setup and use DatX data store within Angular project and how to use it in tests. If you are not familiar with it yet, resources below can help.
+
+## Resources
+
+You can read up on DatX at official [docs site](https://datx.dev/docs/getting-started/installation). Infinum also published article [working with JSON API](https://infinum.com/the-capsized-eight/working-with-JSON-API) with examples.
+
+## Why use DatX
+
+Apart from providing possible single source of truth for any shared data with a basic querying and referencing support, it also provides options to extend any entity with mixins such as [withActions](https://datx.dev/docs/mixins/with-actions), [withMeta](https://datx.dev/docs/mixins/with-meta), [withPatches](https://datx.dev/docs/mixins/with-patches), which enhance entities with usefull utility functions, or network related ones like [withNetwork](https://datx.dev/docs/mixins/with-network), [jsonapi](https://datx.dev/docs/mixins/jsonapi) or, useful in Angular's case, [jsonapiAngular](https://datx.dev/docs/mixins/jsonapi-angular). Especially with [jsonapi](https://datx.dev/docs/mixins/jsonapi) and/or [jsonapiAngular](https://datx.dev/docs/mixins/jsonapi-angular), interacting with [JSONAPI standard](https://jsonapi.org/) compliat backend can be much easier then building requests manually and more managable than building API related code with some sort of generator.
+
+## Dependencies
+Let's start by installing all DatX related dependencies we will need:
+```
+npm i --save @datx/core @datx/jsonapi @datx/jsonapi-angular mobx
```

Comment:
With angular, we usually avoid using mobx since it's an overhead and it doesn't play nice (at least not out of the box) with rxjs.
Because of this, we need the noop.js file you might have noticed in the project (and referenced in angular.json) and the `@datx/core/disable-mobx` import in the app.

Reply:
Expanded on that with a3f71e1